src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -237,8 +234,78 @@ private async Task UpdateJobRecordAsync(CancellationToken cancellationToken)
                 queryParametersList.Add(Tuple.Create(KnownQueryParameterNames.Type, _exportJobRecord.ResourceType));
             }
 
+            var filteredResources = new HashSet<string>();
+            if (_exportJobRecord.Filters != null)
+            {
+                foreach (var filter in _exportJobRecord.Filters)
+                {
+                    filteredResources.Add(filter.ResourceType);
+                }
+            }
+
             IAnonymizer anonymizer = IsAnonymizedExportJob() ? await CreateAnonymizerAsync(cancellationToken) : null;
 
+            if (!progress.RunFilteredSearches)
```

Comment:
nit: The name `RunFilteredSearches` seems to convey that we should run filtered searches when it is `true`. But we do the opposite here :). Do you think renaming it to `FilteredSearchesComplete` or something similar can be better?